pkg/model/project.go

```diff
@@ -427,6 +404,59 @@ func (p *Project) GetAllUserGroups() []*ProjectUserGroup {
 	return all
 }
 
+// IsAssigned checks whether the team is already assigned as built-in roles.
+func (p *ProjectRBACConfig) IsAssigned(team string) bool {
```

Comment:
What is the difference between this function and `HasRBACRole`?
IMO, they are the same and we should merge this to `HasRBACRole` function to be simple.
WDYT?

Reply:
There is a difference between `IsAssigned` and `HasRBACRole`.
`IsAssigned` method of the `ProjectRBAConfig` checks whether the team was already assigned some built-in role.
But I think merging `HasUserGroup` and `IsAssigned` is a good idea, so let me apply it.